base/commandLine.cpp

```diff
@@ -455,7 +458,7 @@ Common::String parseCommandLine(Common::StringMap &settings, int argc, const cha
 				// FIXME: Need to document this.
 				// TODO: Make the argument optional. If no argument is given, list all saved games
 				// for all configured targets.
-				return "list-saves";
```

Comment:
For consistency you should probably check that `command` is empty before setting it here and if it is not display an error as done in the DO_COMMAND.